src/test/java/system_tests/TrialSeleniumTest.java

```diff
@@ -0,0 +1,67 @@
+package system_tests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static system_tests.HttpHandler.postRequest;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class TrialSeleniumTest {
+    private static WebDriver driver;
+
+    @BeforeAll
+    static void setUp(){
+        //TODO set it up for mac/linux: https://stackoverflow.com/questions/228477/how-do-i-programmatically-determine-operating-system-in-java
+        System.setProperty("webdriver.chrome.driver","src\\test\\java\\system_tests\\chromedriver.exe");
+        driver = new ChromeDriver();
```

Comment:
I did a little reasearch on the WebDriver and it seems that Selenium is pushing for the use of `RemoteWebDriver`. In which you start a server to act as your driver instead of using the chromedriver. Then you can connect to that server here using `RemoteWebDriver`. I think that could solve our compatibility issues on running the system tests on different platforms. Here's the documentation I was able to find [https://www.selenium.dev/documentation/en/remote_webdriver/](https://www.selenium.dev/documentation/en/remote_webdriver/), [https://github.com/SeleniumHQ/selenium/wiki/RemoteWebDriver](https://github.com/SeleniumHQ/selenium/wiki/RemoteWebDriver), and [https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver](https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver)

Reply:
Good idea. I believe we need to set up selenium grid (https://www.guru99.com/introduction-to-selenium-grid.html) which enables you to run selenium tests remotely. I suggest opening another issue for setting up the selenium grid.